files/riotgear/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "riotgear",
+  "repo": "RiotGear/rg",
+  "files": {
+    "basePath": "dist/"
```

Comment:
Can you add an `include` parameter and include only the min file? The unminified file is not required in production